transcom/mymove
# B 20529 int add save retrieve and filter capability for assigned column sc

## [B-20529](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20529)

## Summary
Adds in Assigned User Column on Service Counselors Queue with the ability to filter, sort, and assign moves to a Service Counselor.

### How to test as SC
1. Log in as a Service Counselor
2. Click on the dropdown of any move under Assigned, and select a user from the dropdown
3. Repeat step 2 for another user
4. Refresh the page
5. Enter the name of the user that was assigned for either of the moves in the Assigned User column header
6. There should be a pill off to the left of the top of the table with the name of the user filtered and an 'X', only moves that are assigned to the user entered into Assigned User as a filter should display.
7. Click the 'X' button for the filter to remove this filter. After doing so, all moves should show regardless of assigned
8. Click on the Assigned Column to sort in Ascending order, moves should show based on the name of who's assigned
9. Click again to show assigned users in Descending order, moves should show like before but in descending order
10. Click once more to show assigned moves in unordered mode.
